routes/frontend/portal.php

```diff
@@ -88,6 +91,21 @@
 
             Route::get('/media/{media}/{filename}', function (Media $media) {
                 return $media;
-            });
+            })->name('media');
+
+            Route::any('/media/download-multiple', function (Request $request) {
```

Comment:
Why use callback and not run it through a controller?

Reply:
√